### PR TITLE
I have done..!

### PR DESCRIPTION
**Pull request by ankitjain28may**

### CODE REVIEW

**Comment by aakarshgoel96** [comment]
@ankitjain28may include file connectvars.php for connection otherwise we have change db connection parameters in every php file.

**Comment by ankitjain28may** [comment]
But it is being added by you already.

File: MainHospital/Views/submit.php [hunk 1]
```diff
@@ -10,19 +11,17 @@
...
   header('Location: ' . $home_url);
   }
   if (isset($_SESSION['usermain'])) {
 
   }
 $id = $_GET['id'];
-require_once("connectvars.php");
 
-$dbc = mysqli_connect(DB_HOST, DB_USER, DB_PASSWORD, DB_NAME)
+$dbc = mysqli_connect('localhost','root','','healthcare')
```
Comment (aayusharora) [inline MainHospital/Views/submit.php]: @ankitjain28may  , @aakarshgoel96  I think its coming from the config file , can not write directly.

File: MainHospital/Views/submit.php [hunk 2]
```diff
@@ -48,13 +47,11 @@
            
 
             // Write the data to the database
-            $query1 = "UPDATE patient_file SET sreport='$sreport',status='Active',comment='$comment',d_name='$dname',ddept='$ddept',timestamp=NOW() WHERE p_id='$id' ";
+            $query1 = "INSERT INTO doc_report(p_id,sreport,comment,d_name,ddept,timestamp) VALUES ( '$id', '$sreport','$comment','$dname','$ddept',NOW())";
```
Comment (aayusharora) [inline MainHospital/Views/submit.php]: @ankitjain28may  , where is the UPDATE patient_file code gone ?

File: PrimaryHospital/Views/logout.php [hunk 3]
```diff
@@ -19,6 +19,6 @@
   setcookie('username', '', time() - 3600);
 
   // Redirect to the home page
-  $home_url = 'http://' . $_SERVER['HTTP_HOST'] . dirname($_SERVER['PHP_SELF']) . '/#';
+  $home_url = 'http://' . $_SERVER['HTTP_HOST'] . dirname($_SERVER['PHP_SELF']) . '../../../index.html';
   header('Location: ' . $home_url);
```
Comment (aayusharora) [inline PrimaryHospital/Views/logout.php]: @ankitjain28may  Is it working ?

File: PrimaryHospital/Views/submitp.php [hunk 4]
```diff
@@ -1,150 +1,150 @@
...
+    if (isset($_COOKIE['username'])) {
+      $_SESSION['username'] = $_COOKIE['username'];
+    }
+  }
+   if((!isset($_SESSION['username'])) && (!isset($_COOKIE['username'])))
+  {
+    $home_url = 'http://' . $_SERVER['HTTP_HOST'] . dirname($_SERVER['PHP_SELF']) . '../../#/primary';
+  }
+  ?>
+
```
Comment (aayusharora) [inline PrimaryHospital/Views/submitp.php]: @aakarshgoel96 , @ankitjain28may , connectvars.php should be included

**Comment by ankitjain28may** [comment]
Ok, i have the previous code.. this might be the problem.. i wil check it out..!

File: PrimaryHospital/Views/submitp.php [hunk 5]
```diff
@@ -1,150 +1,150 @@
...
+      </ul>
+    </div>
+  </div>
+</nav>
+<?php
+  
+    
+$id = $_GET['id'];
+
+$dbc = mysqli_connect('localhost','root','','healthcare')
```
Comment (aayusharora) [inline PrimaryHospital/Views/submitp.php]: @ankitjain28may  , @aakarshgoel96  check this "hardcoded variables " should be removed . use DB_HOST instead

File: PrimaryHospital/Views/tojson.php [hunk 6]
```diff
@@ -10,17 +11,18 @@
  if (isset($_SESSION['username'])) {
  $name=$_SESSION['username'];
  }
-$dbc = mysqli_connect(DB_HOST, DB_USER, DB_PASSWORD, DB_NAME)
-or
-die('error connecting to MySql server');
+$host = "localhost"; 
+$user = "root"; 
+$pass = ""; 
+$database = "healthcare";
```
Comment (aayusharora) [inline PrimaryHospital/Views/tojson.php]: @ankitjain28may Hardcoded variables should not be there ?

File: PrimaryHospital/Views/tojson.php [hunk 7]
```diff
@@ -1,6 +1,7 @@
+
 <?php 
  session_start();
-require_once("connectvars.php");
+
```
Comment (aayusharora) [inline PrimaryHospital/Views/tojson.php]: @ankitjain28may , where is connectvars.php

File: PrimaryHospital/Views/tojson.php [hunk 8]
```diff
@@ -10,17 +11,18 @@
  if (isset($_SESSION['username'])) {
  $name=$_SESSION['username'];
  }
-$dbc = mysqli_connect(DB_HOST, DB_USER, DB_PASSWORD, DB_NAME)
-or
-die('error connecting to MySql server');
+$host = "localhost";
```
Comment (aayusharora) [inline PrimaryHospital/Views/tojson.php]: @aakarshgoel96  , check this please

File: PrimaryHospital/Views/tojson.php [hunk 9]
```diff
@@ -10,17 +11,18 @@
...
-$data = mysqli_query($dbc, $query);
+$linkID = mysql_connect($host, $user, $pass) or die("Could not connect to host."); 
+mysql_select_db($database, $linkID) or die("Could not find database."); 
+
+$query = mysql_query("SELECT * FROM patient_file WHERE ph_name='$name'");
 $rows = array();
-while($row = mysqli_fetch_assoc($data)) {
+while($row = mysql_fetch_assoc($query)) {
     $rows[] = $row;
 }
```
Comment (aayusharora) [inline PrimaryHospital/Views/tojson.php]: @aakarshgoel96  , check this please

File: jsontime.php [hunk 10]
```diff
@@ -1,15 +1,16 @@
 <?php 
-require_once("connectvars.php");
-$dbc = mysqli_connect(DB_HOST, DB_USER, DB_PASSWORD, DB_NAME)
-or
-die('error connecting to MySql server');
+$host = "localhost"; 
+$user = "root";
```
Comment (aayusharora) [inline jsontime.php]: @aakarshgoel96  , @ankitjain28may  , take it from connectvars.php

File: jsontime.php [hunk 11]
```diff
@@ -1,15 +1,16 @@
...
-$query = "SELECT * FROM patient_file WHERE status='Active' ORDER BY timestamp DESC";
-$data = mysqli_query($dbc, $query);
+$linkID = mysql_connect($host, $user, $pass) or die("Could not connect to host."); 
+mysql_select_db($database, $linkID) or die("Could not find database."); 
+
+$query = mysql_query("SELECT * FROM patient_file WHERE status='Active' ORDER BY timestamp DESC;");
 $rows = array();
-while($row = mysqli_fetch_assoc($data)) {
+while($row = mysql_fetch_assoc($query)) {
     $rows[] = $row;
```
Comment (aayusharora) [inline jsontime.php]: @aakarshgoel96 , check this

**Comment by aayusharora** [comment]
@ankitjain28may  , please host it to check .. on the local before I can merge .Also see the comments I have made.

**Comment by ankitjain28may** [comment]
ok

**Pull request closed without merging**